src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -21,7 +21,8 @@ namespace Bloom.CollectionTab
 {
 	public partial class LibraryListView : UserControl
 	{
-		private const int ButtonHeight = 110;
+		private const int ButtonHeight = 112;
+		private const int ButtonWidth = 92;
```

Comment:
Height and width increased so that the area available for text will be the same.